File: gluecodium/src/test/resources/smoke/comments/output/android/com/example/smoke/PlatformCommentsLineBreaks.java
/*
 *
 */
package com.example.smoke;
import com.example.NativeBase;
/**
 * <p>Text middle
 * line break
 */
public final class PlatformCommentsLineBreaks extends NativeBase {
    /**
     * For internal use only.
     * @hidden
     */
    protected PlatformCommentsLineBreaks(final long nativeHandle, final Object dummy) {
        super(nativeHandle, new Disposer() {
            @Override
            public void disposeNative(long handle) {
                disposeNativeHandle(handle);
            }
        });
    }
    private static native void disposeNativeHandle(long nativeHandle);
}

File: gluecodium/src/test/resources/smoke/comments/output/android-kotlin/com/example/smoke/PlatformCommentsLineBreaks.kt
/*

 *
 */

@file:JvmName("PlatformCommentsLineBreaksExtensions")

package com.example.smoke

import com.example.NativeBase

/**
 * Text
 */
class PlatformCommentsLineBreaks : NativeBase {



    /**
     * For internal use only.
     * @suppress
     * @param nativeHandle The handle to resources on C++ side.
     * @param tag Tag used by callers to avoid overload resolution problems.
     */
    protected constructor(nativeHandle: Long, @Suppress("UNUSED_PARAMETER") tag: Any?)
        : super(nativeHandle, { disposeNativeHandle(it) }) {}







    companion object {
        @JvmStatic private external fun disposeNativeHandle(nativeHandle: Long)
    }
}

